pkg/platform/platform.go

```diff
@@ -137,13 +137,15 @@ type Platform interface {
 	GetBootFiles() [][2]string
 	GetBootloader() Bootloader
 	GetFIPSMenu() bool
+	GetDefaultMenu() int
```

Comment:
I'm not really sure if the default boot menu should be set / customized via the `Platform` definition :thinking: Having it in the `ImageConfig` and specified per image type may make more sense.